src/docs/development/tools/android-studio.md

```diff
@@ -87,88 +87,78 @@ The Dart plugin performs code analysis that enables:
 
 ## Running and debugging
 
+{{site.alert.note}}
+  You can debug your app in a few ways.
+
+  * Using [DevTools][], a suite of debugging and profiling
+    tools that run in a browser
+    _and include the Flutter inspector_.
+    DevTools replaces the previous browser-based profiling
+    tool, Observatory.
+  * Using Android Studio's (or IntelliJ's) built-in debugging
+    features, such as the ability to set breakpoints.
+  * Using the Flutter inspector, directly available in
+    Android Studio and IntelliJ.
+
+  The instructions below describe features available in Android
+  Studio and IntelliJ. For information on launching DevTools,
+  see [Running DevTools from Android Studio][] in the
```

Comment:
Should all references to "from Android Studio" actually say "from Android Studio / IntelliJ"? It seems like we mention both IDEs elsewhere in the docs (Ex. Tools & techniques -> Android Studio / IntelliJ)